swirl/build.gradle

```diff
@@ -24,9 +18,8 @@ android {
 }
 
 dependencies {
-  compile 'com.android.support:support-annotations:27.0.2'
-  compile 'com.android.support:support-vector-drawable:27.0.2'
-  compile 'com.android.support:animated-vector-drawable:27.0.2'
+  compileOnly "com.android.support:support-annotations:${versions.supportLibrary}"
```

Comment:
AVD transitively depends on annotations so you're always going to take an implementation dependency on annotations no matter what you do here.

Reply:
updated to `implementation`